pyfans/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+pybind11_add_module(PyFANS micro.hpp micro.cpp)
+target_link_libraries(PyFANS PRIVATE FANS::FANS)
+
+add_custom_command(
+        TARGET PyFANS
+        POST_BUILD
+        COMMAND ${CMAKE_COMMAND} -E create_symlink
+        $<TARGET_FILE:PyFANS>
+        ${CMAKE_CURRENT_SOURCE_DIR}/../test/test_pyfans/$<TARGET_FILE_NAME:PyFANS>
+        COMMENT "Create a symlink for FANS python bindings to ${CMAKE_CURRENT_SOURCE_DIR}/../test/"
+)
+
+option(USE_MPI "Use MPI" OFF)
+
+if (USE_MPI)
+        add_definitions(-DUSE_MPI)
```

Comment:
Is this compiler definition only relevant for pyFANS or for the whole project? In any case, it's recommended to use target specific definitions where possible. I suggest to change this to `target_compile_definitions(<target> <INTERFACE|PUBLIC|PRIVATE> ...)`.

Reply:
I am actually not sure. The way pyFANS will be used, it should not do anything with MPI. The MPI part is handled by the Micro Manager. pyFANS itself has no notion of MPI.